src/backend/commands/player/kick/multi_kick.cpp

```diff
@@ -18,12 +18,12 @@ namespace big
 		virtual void execute(player_ptr player, const command_arguments& _args, const std::shared_ptr<command_context> ctx) override
 		{
 			if (g_player_service->get_self()->is_host())
-				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {}),
-				    dynamic_cast<player_command*>(command::get("hostkick"_J))->call(player, {});
-
-			dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
-			script::get_current()->yield(700ms);
+				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {});
 
+			if (player->is_host()) {
```

Comment:
You should still check player. You just don't need the is_valid() check.

`if (player && player->is_host())`